tests/test_well/test_blockedwells.py

```diff
@@ -18,8 +19,7 @@
 @pytest.fixture(name="testblockedwells")
 def fixture_testblockedwells(testpath):
     well_files = [join(testpath, "wells", "reek", "1", "OP_1.bw")]
-    bws = BlockedWells()
-    bws.from_files(well_files)
+    bws = xtgeo.blockedwells_from_files(well_files)
```

Comment:
How good is the test coverage on `BlockedWells`?

Reply:
About 50% so lower than most other xtgeo code.

Reply:
But there is only one place you had to update this pattern?

Reply:
Which pattern?

Reply:
from `BlockedWells` to `xtgeo.blockedwells_from_files`

Reply:
No, we just changed it here for now to have some test coverage, we need to remove deprecated useage in tests separately.